### ISSUE TITLE
Was the method setInsecure removed ?

### ISSUE THREAD

**Issue by ansel2000** [electronicsguy/HTTPSRedirect#2]
I keep getting this error

error: 'class HTTPSRedirect' has no member named 'setInsecure'
     client->setInsecure();

**Comment by electronicsguy** [comment]
Yes, it doesn't have that member function, so you shouldn't be using it. What are you trying to do?
